libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/camera/CameraComponent.kt

```diff
@@ -14,133 +11,117 @@ import com.mapbox.navigation.dropin.extensions.flowRoutesUpdated
 import com.mapbox.navigation.dropin.lifecycle.UIComponent
 import com.mapbox.navigation.ui.maps.camera.NavigationCamera
 import com.mapbox.navigation.ui.maps.camera.data.MapboxNavigationViewportDataSource
+import com.mapbox.navigation.ui.maps.camera.lifecycle.NavigationBasicGesturesHandler
 import com.mapbox.navigation.ui.maps.camera.state.NavigationCameraState
 import kotlinx.coroutines.flow.collect
-import kotlinx.coroutines.flow.filterNotNull
 import kotlinx.coroutines.launch
 
 internal class CameraComponent constructor(
     private val mapView: MapView,
+    private val navigationCamera: NavigationCamera,
+    private val viewportDataSource: MapboxNavigationViewportDataSource,
     private val locationViewModel: LocationViewModel,
     private val cameraViewModel: CameraViewModel,
 ) : UIComponent() {
 
-    private var isLocationInitialized = false
-    private lateinit var navigationCamera: NavigationCamera
-    private lateinit var viewportDataSource: MapboxNavigationViewportDataSource
-
-    private val onMoveListener = object : OnMoveListener {
-        override fun onMove(detector: MoveGestureDetector): Boolean {
-            return false
-        }
+    // TODO: Remove or improve to calculate the viewport size at runtime
+    private val overviewEdgeInsets = EdgeInsets(
+        150.0 * mapView.context.resources.displayMetrics.density,
+        200.0 * mapView.context.resources.displayMetrics.density,
+        200.0 * mapView.context.resources.displayMetrics.density,
+        200.0 * mapView.context.resources.displayMetrics.density,
+    )
+    private val followingEdgeInsets = EdgeInsets(
+        0.0 * mapView.context.resources.displayMetrics.density,
+        0.0 * mapView.context.resources.displayMetrics.density,
+        200.0 * mapView.context.resources.displayMetrics.density,
+        0.0 * mapView.context.resources.displayMetrics.density,
```

Comment:
You don't want to use `Utils.dpToPx()` for this?

Reply:
We could do it, but I did not hold too much opinion here since this edgeInsets are not gonna have hardcoded values in the first place and will change at runtime. I put it temporarily to make the camera functioning look nice.

Reply:
Not a strong opinion from me on this one. But I also prefer `Utils.dpToPx()`, is there an issue with it? Seems like it's easier to mock for unit tests and `Utils.dpToPx` does not need the `MapView`. We remove a dependency

Reply:
Done